PHPCompatibility/Docs/Classes/NewConstVisibilityStandard.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+<documentation title="New constant visibility">
+    <standard>
+    <![CDATA[
+    Class constants can be declared with visibility since PHP 7.1.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Cross-version compatible: not using a visibility modifier.">
+        <![CDATA[
+class Foo {
+    const BAR = 10;
+}
+        ]]>
+        </code>
+        <code title="PHP >= 7.1: using a visibility modifier.">
+        <![CDATA[
+class Foo {
+    <em>protected</em> const BAR = 10;
+}
+
+        ]]>
```

Comment:
The HTML tag appears literally in markdown docs, and I believe that there is a trailing newline. I recommend avoiding markdown in code samples. Other than that, everything looks good on the docs side.

```suggestion
        <![CDATA[
class Foo {
    protected const BAR = 10;
}
        ]]>
```

Reply:
I've fixed up the trailing new line. Thanks for catching that!

Regarding the `<em>` markers: see my comments in the other PR: https://github.com/PHPCompatibility/PHPCompatibility/pull/1317#discussion_r825063690